### PR TITLE
Filipino (Tagalog) Localization #1

### PR DESCRIPTION
**Pull request by burdagay**
Translated the following files to Filipino (Tagalog):
1. tl-tl/sc/api.md
2. tl-tl/sc/fw.md
3. tl-tl/sc/tutorial.md

### CODE REVIEW

File: tl-tl/sc/api.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# Smart Contract API Reference
+# Reperensiya para sa API ng Smart Contract
 
-The NEO Smart Contract API expands the capabilities of the Smart Contract to access the blockchain data, manipulate persistent storage, and query the execution environment. It is part of the Neo Virtual Machine (NeoVM) Interoperability Service Layer.
+Ang API ng NEO Smart Contract ay nagpapalawak ng kapabilidad ng Smart Contract na makuha ang blockchain na datos, imanipula ang di mawawalang storage, at umusisa sa eskekusyon na environment. Ito ay bahagi ng Neo Virtual Machine (NeoVM) Interoperability Service Layer.
```
Comment (severinolorillajr) [inline tl-tl/sc/api.md]: eskekusyon to pagganap

File: tl-tl/sc/api.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+Para sa karagdagang impormasyon kung paano gamitin ang framework para sa madaling pagtawag ng mga API sa mga mataas na lebel na mga lengguwahe, tingnan ang [Ants Smart Contracts Framework](fw.md).
 
 
-Namespace | description |
+Namespace | Deskripsyon |
 | ----------------------------- | ---------------------------------------- |
-| [Neo](api/neo.md) | The Neo namespace is the API provided by the NEO blockchain, providing a way to access the blockchain data and manipulate the persistent store. |
-| [System](api/system.md) | The System namespace is the API provided by the Smart Contract Execution Engine, providing access to the execution environment that accesses the smart contract. |
+| [Neo](api/neo.md) | Ang Neo namespace ay ang API na binibigay ng NEO blockchain, na nagbibigay ng paraan upang makuha ang blockchain na datos at imanipula ang di nawawalang store. |
+| [System](api/system.md) | Ang System namespace ay ang API na binibigay ng Smart Contract Execution Engine, ito ay nagbibigay ng paraan para makuha ang eksekyusyon na environment na gumagamit sa smart contract. |
```
Comment (severinolorillajr) [inline tl-tl/sc/api.md]: eksekusyon to pagganap

File: tl-tl/sc/tutorial.md [hunk 3]
```diff
@@ -55,9 +55,9 @@ public static bool Main ()
 }
 ```
 
-Here the return value of the contract is always true, indicating that anyone can spend the contract address of the assets (can be understood as money).
+Dito ang isinasauli na halaga ng kontrata ay parating true, nagpapahiwatig na kahit sino ay pwedeng gumastos ng contract address ng mga asset (pwedeng isipin na pera).
 
-The NEO wallet client has a function for deleting an asset. When you delete an asset, the asset is sent to a specified address being the contract address generated by the above smart contract. Anyone can spend the assets in the address. The assets in the address are assets that others do not want.
+Ay NEO wallet na client ay mayroong punsyon para tumanggal ng isang asset. Kung tatanggal ka nang asset, ang asset ay ipapadala patungo sa tinukoy na address na nagrerepresenta sa nagawang address ng nasa itaas na mataliong kontrata. Kahit sino ay pwedeng gumastos ng asset na nasa address. Ang mga asset na nasa address ay mga asset na hindi gusto ng iba.
```
Comment (severinolorillajr) [inline tl-tl/sc/tutorial.md]: Ang Neo wallet

**Comment by severinolorillajr** [comment]
Hi @burdagay , just a few minor update and we can merge this.

File: tl-tl/sc/api.md (hunk 1 above)
Comment (burdagay) [reply tl-tl/sc/api.md]: This is noted. It has now been fixed.

File: tl-tl/sc/api.md (hunk 2 above)
Comment (burdagay) [reply tl-tl/sc/api.md]: This is noted. It has now been fixed.

File: tl-tl/sc/tutorial.md (hunk 3 above)
Comment (burdagay) [reply tl-tl/sc/tutorial.md]: This is noted. It has now been fixed.

**Comment by burdagay** [comment]
Hi @severinolorillajr I already updated this PR. Please recheck. Thanks.

**Comment by dbucci24** [comment]
@severinolor hey, i would like to translate this project into Spanish, is this done or is it still necessary?

**Comment by severinolorillajr** [comment]
@dbucci24 , check with the project owner.
This forked repo is specific for Filipino/Tagalog translation.
Thanks.

**Pull request merged**